internal/host/monitor_test.go

```diff
@@ -130,11 +135,13 @@ var _ = Describe("TestHostMonitoring", func() {
 		mockEvents.EXPECT().
 			AddEvent(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).
 			AnyTimes()
+		mockMetrics = metrics.NewMockAPI(ctrl)
+		mockMetrics.EXPECT().HostValidationFailed(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).AnyTimes()
```

Comment:
Usually i don't like the use of any times, it can hide bugs, any times is zero and above so if it's not called at all it will still pass.

Reply:
Fixed.